### PR TITLE
Add aws release 16.2.0

### PR DESCRIPTION
**Pull request by whites11**

### CODE REVIEW

**Comment by whites11** [comment]
cncf test is green

**Comment by njuettner** [comment]
Can we include aws-ebs-csi-driver 2.8.0? I created a request for including it https://github.com/giantswarm/releases/pull/884, this would enable us to start collecting metrics.

**Comment by whites11** [comment]
> Can we include aws-ebs-csi-driver 2.8.0? I created a request for including it #884, this would enable us to start collecting metrics.

does it require me to start testing from scratch again?

**Comment by whites11** [comment]
> Can we include aws-ebs-csi-driver 2.8.0? I created a request for including it #884, this would enable us to start collecting metrics.

bumped, running e2e tests again

**Comment by njuettner** [comment]
Sorry I saw you comment to late 🤦🏻. Actually it should only pass the aws tests, we're testing creating EBS volumes there.

File: aws/v16.2.0/release.yaml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  annotations:
+    giantswarm.io/docs: https://docs.giantswarm.io/reference/cp-k8s-api/releases.release.giantswarm.io/
+  creationTimestamp: null
+  name: v16.2.0
+spec:
+  apps:
+  - componentVersion: 1.4.0
```
Comment (njuettner) [inline aws/v16.2.0/release.yaml:12]: The componentVersion also needs a bump
```suggestion
  - componentVersion: 1.5.0
```

**Comment by whites11** [comment]
aws and aws-china tests green

File: aws/v16.2.0/release.diff [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+apiVersion: release.giantswarm.io/v1alpha1                         apiVersion: release.giantswarm.io/v1alpha1
+kind: Release                                                      kind: Release
+metadata:                                                          metadata:
+  annotations:                                                       annotations:
+    giantswarm.io/docs: https://docs.giantswarm.io/reference/cp        giantswarm.io/docs: https://docs.giantswarm.io/reference/cp
+  creationTimestamp: null                                            creationTimestamp: null
+  name: v16.1.0                                                 |    name: v16.2.0
+spec:                                                              spec:
+  apps:                                                              apps:
+  - componentVersion: 1.4.0                                          - componentVersion: 1.4.0
```
Comment (njuettner) [inline aws/v16.2.0/release.diff:12]: sorry again. nit, you also need to update the component version here 🙂

Comment (whites11) [reply aws/v16.2.0/release.diff]: done

File: aws/v16.2.0/README.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
+# :zap: Giant Swarm Release v16.2.0 for AWS :zap:
+
+This is a security release featuring the latest Flatcar Linux version as well as an updated version of the Giant Swarm applications.
```
Comment (alex-dabija) [inline aws/v16.2.0/README.md:3]:
```suggestion
This is a security release featuring the latest Flatcar Linux version as well as various updates to Giant Swarm applications.
```

File: aws/v16.2.0/announcement.md [hunk 4]
```diff
@@ -0,0 +1 @@
+**Workload cluster release v16.2.0 for AWS is available**. This is a security release featuring the latest Flatcar Linux version as well as an updated version of the Giant Swarm applications. Further details can be found in the [release notes](https://docs.giantswarm.io/changes/workload-cluster-releases-aws/releases/aws-v16.2.0/).
```
Comment (alex-dabija) [inline aws/v16.2.0/announcement.md:1]:
```suggestion
**Workload cluster release v16.2.0 for AWS is available**. This is a security release featuring the latest Flatcar Linux version as well as various updates to Giant Swarm applications. Further details can be found in the [release notes](https://docs.giantswarm.io/changes/workload-cluster-releases-aws/releases/aws-v16.2.0/).
```

**Review by alex-dabija — APPROVED** [review]

**Pull request merged**